server/helpers/getScopes.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const PermissionConfigTable = require('../../permission-config.json');
+const DefaultRoles = require('./getDefaultRoles');
+
+module.exports = (path, method, scope) => {
```

Comment:
Since scope parameter is supposed to be an array please change the name to scopes and add s block comment explaining what this function does.